indexer/disk-saver/src/error.rs

```diff
@@ -0,0 +1,19 @@
+use {grug_types::StdError, std::path::PathBuf};
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error(transparent)]
+    Std(#[from] StdError),
+
+    #[error(transparent)]
+    Io(#[from] std::io::Error),
+
+    #[error(transparent)]
+    Persist(#[from] tempfile::PersistError),
+
+    #[error(transparent)]
+    Lzma(#[from] lzma_rs::error::Error),
+
+    #[error("File already exists: {0}")]
```

Comment:
Do you uppercase the error message. See [rust style guide](https://rustc-dev-guide.rust-lang.org/diagnostics.html#diagnostic-output-style-guide:~:text=Error%2C%20Warning%2C%20Note%2C%20and%20Help%20messages%20start%20with%20a%20lowercase%20letter%20and%20do%20not%20end%20with%20punctuation.)